### ISSUE TITLE
Users removed from lxd group able to access "lxc remote" commands

### ISSUE THREAD

**Issue by mrheroux** [lxc/lxd#5033]
When a user is removed from the lxd group (deluser test lxd") they can no longer stop and start containers for security reasons. However, it looks as though users removed from the lxd group still have access to the lxc remote commands. This seems like a security issue.

![image](https://user-images.githubusercontent.com/19890291/45562175-9a764180-b817-11e8-83fb-8c909796d45e.png)
Running LXC version 3.01

Is there something I'm missing?

**Comment by mrheroux** [comment]
It looks like they cannot "lxc remote remove container" and their "lxc remote list" does not include the hosts added by users in the lxd group. Also, they cannot actually add a host. Still seems odd that its available. I would think they wouldn't be able to execute the commands.

![image](https://user-images.githubusercontent.com/19890291/45563980-fabbb200-b81c-11e8-89e2-4d43070fe18f.png)

**Comment by mrheroux** [comment]
Yeah a person removed from the LXD group can add remote host which are not available to the user inside the lxd group. Sounds like an issue. 
![image](https://user-images.githubusercontent.com/19890291/45564183-a82ec580-b81d-11e8-9b80-e584c09c80a8.png)

**Comment by stgraber** [comment]
On snap installs, you can look at `~/snap/lxd/current/.config/lxc/config.yml`, on non-snap system, it's at `~/.config/lxc/config.yml`. This contains all your remote configurations and aliases. The directory that file is in also includes the client certificate that the CLI generates as well as the expected public certificate for the remote servers it interacts with over HTTPs.
